build_test/lib/src/matchers.dart

```diff
@@ -5,10 +5,18 @@ import 'package:test/test.dart';
 
 import 'package:build/build.dart';
 
+/// Matches that the provided instance is a [AssetNotFoundException].
```

Comment:
[nit] This reads strangely to me. Maybe "Matches instances of [AssetNotFoundException]."

Reply:
Done.